lib/confirm-controller.js

```diff
@@ -1,61 +1,129 @@
 'use strict';
 
-const BaseController = require('./base-controller');
 const _ = require('lodash');
+const i18nLookup = require('i18n-lookup');
+const Mustache = require('mustache');
```

Comment:
Mustache is a dependency of Hogan, so if hogan is used elsewhere Mustache is included